spec/order_checker_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'order_checker'
+
+describe OrderChecker do
+
+  menu = [
+      { dish: "Sea salt & caramel",      price: 3.5 },
+      { dish: "Chilli & chocolate",      price: 3.5 },
```

Comment:
You probably don't need this much test data - unit tests should have just enough data to test the code.